plugins/outputs/http/README.md

```diff
@@ -65,6 +65,15 @@ batch format by default.
   #   # Should be set manually to "application/json" for json data_format
   #   Content-Type = "text/plain; charset=utf-8"
 
+  ## MaxIdleConns controls the maximum number of idle (keep-alive)
+  ## connections across all hosts. Zero means no limit.
+  # max_idle_conn = 0
+
+  ## MaxIdleConnsPerHost, if non-zero, controls the maximum idle
+  ## (keep-alive) connections to keep per-host. If zero,
+  ## DefaultMaxIdleConnsPerHost is used(2).
+  # max_idle_conn_per_host = 0
```

Comment:
Similar to the above comment, this should remain at the previous default. Per #10953, you said it was 2. While your comment appears to specify that setting it to zero, will actually set it to 2, I would rather us be explicit about the actual behavior.